components/scheduler/refresh-data.md

```diff
@@ -13,9 +13,125 @@ position: 40
 @[template](/_contentTemplates/common/observable-data.md#intro)
 
 In this article:
+- [Rebind Method](#rebind-method)
 - [Observable Data](#observable-data)
 - [New Collection Reference](#new-collection-reference)
 
+## Rebind Method
+
+You can refresh the Scheduler data by using the Rebind method exposed to the reference of the TelerikScheduler.
```

Comment:
Some more info on when this is needed and when not. Also link to http://localhost/blazor-ui/common-features/data-binding/overview#refresh-data. You may use the template similar to https://github.com/telerik/blazor-docs/pull/1129/files#diff-8e3e1b52deeaa116313bff9e1941eb5df7ed9a71c14e75e2fb5d67d2b7e14cffR22